index.html

```diff
@@ -79,7 +95,79 @@
           </div>
         </div>
 
-        <div data-view="settings" class="hidden"></div>
+        <div data-view="settings" class="hidden">
+          <div class="row">
+            <div class="column-full">
+              <div class="settings-text">Settings</div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-two-thirds">
+              <form>
+                <div>
+                  <label for="category">Select Category:</label>
+                </div>
+                <div>
+                  <select id="category" name="category">
+                    <option value="">Select</option>
+                    <option value="9">General Knowledge</option>
+                    <option value="10">Entertainment: Books</option>
+                    <option value="11">Entertainment: Film</option>
+                    <option value="12">Entertainment: Music</option>
+                    <option value="13">
+                      Entertainment: Musicals &amp; Theatres
+                    </option>
+                    <option value="14">Entertainment: Television</option>
+                    <option value="15">Entertainment: Video Games</option>
+                    <option value="16">Entertainment: Board Games</option>
+                    <option value="17">Science &amp; Nature</option>
+                    <option value="18">Science: Computers</option>
+                    <option value="19">Science: Mathematics</option>
+                    <option value="20">Mythology</option>
+                    <option value="21">Sports</option>
+                    <option value="22">Geography</option>
+                    <option value="23">History</option>
+                    <option value="24">Politics</option>
+                    <option value="25">Art</option>
+                    <option value="26">Celebrities</option>
+                    <option value="27">Animals</option>
+                    <option value="28">Vehicles</option>
+                    <option value="29">Entertainment: Comics</option>
+                    <option value="30">Science: Gadgets</option>
+                    <option value="31">
+                      Entertainment: Japanese Anime &amp; Manga
+                    </option>
+                    <option value="32">
+                      Entertainment: Cartoon &amp; Animations
+                    </option>
```

Comment:
Are you able to extract this information from the API rather than hard coding yourself? What if the API adds more category options?

Reply:
Yes they provide a way to do this:
Category Lookup: Returns the entire list of categories and ids in the database.
https://opentdb.com/api_category.php